src/rpc/blockchain.h

```diff
@@ -57,8 +57,11 @@ void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fInclud
 void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry, bool include_hex = true, int serialize_flags = 0, const CTxUndo* txundo = nullptr);
 
 NodeContext& EnsureNodeContext(const std::any& context);
+CTxMemPool& EnsureMemPool(const NodeContext& node);
 CTxMemPool& EnsureMemPool(const std::any& context);
+ChainstateManager& EnsureChainman(const NodeContext& node);
 ChainstateManager& EnsureChainman(const std::any& context);
+CBlockPolicyEstimator& EnsureFeeEstimator(const NodeContext& node);
 CBlockPolicyEstimator& EnsureFeeEstimator(const std::any& context);
```

Comment:
In commit "rpc: Add alt Ensure* functions acepting NodeContext" (306b1cd3eeb2502904ed4698646d2c86d028aad2)

Caramba. I think this commit is ok, but it is a footgun to have overloaded functions where one of the functions takes an untyped std::any argument, because someone could easily try to call the typed overload, and fail because the type is a little off (const instead of nonconst, pointer instead of reference, or just accidentally passing the wrong variable), and the compiler will never catch it because it will fall back to the std::any overload.

I think the new functions are good, but the existing std::any functions should be renamed to EnsureAnyMemPool, EnsureAnyChainman, etc to be more distinct. Or maybe the std::any functions should take JSONRPCRequest arguments instead. Or maybe they can be removed after a lot of usages are removed in the next commit.

Anyway no need to worry about now. Save for bundle √26